gitlab/misc.sh

```diff
@@ -242,6 +275,13 @@ alpine_prepare_vault_job() {
   install_kustomize
 }
 
+alpine_prepare_pluto_job() {
+  alpine_install_pkg findutils git coreutils libc6-compat
+
+  install_kustomize
```

Comment:
Can we split this in 

`alpine_prepare_pluto_job` to install everything but kustomize 
`alpine_prepare_pluto_job_kustomize` to run the previous one + kustmoize - to be used for kustomize repo 
`alpine_prepare_pluto_job_kubecfg` to run the previous one + kubecfg - to be used for k8s-bootstrap

Reply:
No longer needed since it uses the rendered manifests now